test-framework/junit5/src/main/java/io/quarkus/test/junit/nativeimage/NativeBuildOutputExtension.java

```diff
@@ -47,6 +47,14 @@ public void verifyImageMetrics() {
     }
 
     public void verifyImageMetrics(String propertiesFileName) {
+        /*
+         * Allow users to skip this kind of tests by setting env variable QUARKUS_NATIVE_IT_SKIP_VERIFY_IMAGE_METRICS to true
+         */
+        String skipVerifyImageMetrics = System.getenv("QUARKUS_NATIVE_IT_SKIP_VERIFY_IMAGE_METRICS");
+        boolean skip = skipVerifyImageMetrics != null && skipVerifyImageMetrics.equalsIgnoreCase("true");
```

Comment:
Minor nitpicking: you can use Boolean.valueOf here

Reply:
Done, using `Boolean.parseBoolean` though since there is no need for the variable to be a `Boolean` instead of a `boolean`.